doc/operator-capabilities.md

```diff
@@ -94,9 +98,10 @@ The Operator offers the following features related to upgrades:
 
 ## Level 3 - Full Lifecycle
 
-The Operator offers one or more of the following lifecycle management features:
+It should be possible to backup and restore the operand from the operator itself without any additional manual intervention other than triggering these operations. The operand data that should be backed up is any stateful data managed by the operand. You don’t need to backup the CR itself. You do not need to backup the k8s resources created by the operator as the operator should return all resources to the same state if the CR is recreated. If your operator does not already setup the operand with other k8s resilient best practices, this should be completed to achieve this capability level. This includes liveness and readiness probes, multiple replicas, rolling deployment strategies, pod disruption budgets, CPU and memory requests and limits.
```

Comment:
I think we can omit mentioning the backup of the CR but instead advise that the Operator recreates CR as part of the restore from a backup.

Reply:
Thanks @dmesser , I have reworded this a small bit now. I am under the impression that restore would be any operand stateful data. The created CR should be stored somewhere else and re-instantiated by the user, would you agree with this?